packages/gestalt/src/utils/isNotNullish.js

```diff
@@ -0,0 +1,6 @@
+// @flow strict
+
+// $FlowFixMe[unclear-type] The whole point is that we don't know the type of val
+export default function isNotNullish(val: any): boolean {
```

Comment:
Added a "global" util since this is useful anywhere we have props that could be defined with falsy values

Reply:
I try to avoid util functions like these since it's one more level of abstraction. In this case, can we use `!= null` instead of `val !== null && val !== undefined`?

Reply:
I'm a fan of favoring explicit over implicit behavior, hence the two checks instead of using `!=`. But I'll move this function into Text until we prove a more global need